doc/classes/AudioStreamGenerator.xml

```diff
@@ -24,6 +25,39 @@
 		    for i in range(frames_available):
 		        playback.push_frame(Vector2.ONE * sin(phase * TAU))
 		        phase = fmod(phase + increment, 1.0)
+		[/gdscript]
+		[csharp]
+                [Export] public AudioStreamPlayer Player { get; set; }
+
+                private float _pulseHz = 440.0f; // The frequency of the sound wave.
+                private float _sampleHz;
+                private AudioStreamGeneratorPlayback _playback; //Will hold the AudioStreamGeneratorPlayback.
```

Comment:
I would keep the order consistent with GDScript, this should be a 1:1 translation if possible.
If the order used here seems better, the GDScript example can be changed too.